### ISSUE TITLE
systemd-tmpfiles --create issue in CentOS 8 and pSQL

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-backend#600]
```
[sandoche@vps804922 Zonemaster-Backend]$ sudo systemd-tmpfiles --create
[/usr/lib/tmpfiles.d/pesign.conf:1] Line references path below legacy directory /var/run/, updating /var/run/pesign → /run/pesign; please update the tmpfiles.d/ drop-in file accordingly.
[/usr/lib/tmpfiles.d/postgresql.conf:1] Line references path below legacy directory /var/run/, updating /var/run/postgresql → /run/postgresql; please update the tmpfiles.d/ drop-in file accordingly.
```

**Comment by sandoche2k** [comment]
I am stalled in the testing process @mattias-p and @matsduf without an update for the issue.

**Comment by matsduf** [comment]
> I am stalled in the testing process @mattias-p and @matsduf without an update for the issue.

When I search for "Line references path below legacy directory" I get many hits, but I see no solution yet.

**Comment by sandoche2k** [comment]
OK

**Comment by matsduf** [comment]
Fixed by #601

**Comment by sandoche2k** [comment]
Tested and as this issue is fixed by #601
